index.js

```diff
@@ -131,9 +110,13 @@ bot.Dispatcher.on(Events.GATEWAY_READY, () => {
   Commands['initializeTop'].fn(bot, uvClient)
 })
 
+process.on('uncaughtException', (err) => {
+  logger.raven(err.stack || err)
+})
+
 process.on('unhandledRejection', (reason, p) => {
   if (p !== null && reason !== null) {
-    bugsnag.notify(new Error(`Unhandled promise: ${require('util').inspect(p, {depth: 3})}: ${reason}`))
+    logger.raven(new Error(`Unhandled promise: ${require('util').inspect(p, {depth: 3})}: ${reason}`))
```

Comment:
Do a `typeof` check to make sure `p` is not an error already, if it is, send it straight to raven.